src/python/pants/option/global_options.py

```diff
@@ -874,11 +874,14 @@ def register_options(cls, register):
             "tags ('-' prefix).  Useful with ::, to find subsets of targets "
             "(e.g., integration tests.)",
         )
+        ci_var = os.environ.get("CI")
```

Comment:
I suggest just checking the presence of this env variable, not its value.
i.e. `is_ci == "CI" in os.environ`